cpg_workflows/metamist.py

```diff
@@ -89,6 +89,29 @@
 _metamist: Optional['Metamist'] = None
 
 
+def silent_query(query_to_run, query_params: dict | None = None):
```

Comment:
I'm wondering if we could change the name of this. It isn't really a silent query, because if log_metamist is set then it will still be be a very noisy query.

Reply:
That is a fair comment... maybe `gql_query_optional_logging`?